### PR TITLE
Feature/wdp190703 12/rwd topbar i banner

### PR DESCRIPTION
**Pull request by DzesikaTkaczyk**

### CODE REVIEW

File: src/sass/components/_header.scss [hunk 1]
```diff
@@ -223,3 +208,161 @@ header {
...
+            margin-left: 5px;
+          }
+        }
+      }
+
+      .text-right {
+        ul {
+          padding-right: 15px;
+        }
+        ul li {
```
Comment (thisisui) [inline src/sass/components/_header.scss]: to be consistent with other files please use enter before that line and do the same for other parts of the file

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -223,3 +208,161 @@ header {
...
+      .phone-hidden {
+        display: none;
+      }
+
+      .text-left {
+        ul {
+          padding-left: 15px;
+        }
+
+        ul li {
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: to many nesting, you css class instead

File: src/sass/components/_header.scss [hunk 3]
```diff
@@ -223,3 +208,161 @@ header {
...
+          margin-right: 15px;
+          i {
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: add empty line before

File: src/sass/components/_header.scss [hunk 4]
```diff
@@ -223,3 +208,161 @@ header {
...
+
+          i {
+            margin-right: 5px;
+          }
+        }
+      }
+    }
+    .company-claim {
+      position: relative;
+      .center {
```
Comment (matwjablonski) [inline src/sass/components/_header.scss]: add empty line before

Comment (DzesikaTkaczyk) [reply src/sass/components/_header.scss]: poprawilam

Comment (thisisui) [reply src/sass/components/_header.scss]: but did you push it :)

**Pull request merged**